<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stat</class>
 <widget class="QMainWindow" name="Stat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="NumOfVic">
          <property name="font">
           <font>
            <family>DejaVu Sans</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Elvesztegetett idő:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="TimePlayed_value">
          <property name="font">
           <font>
            <family>DejaVu Sans</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>N/A</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="TimePlayed">
          <property name="font">
           <font>
            <family>DejaVu Sans</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Győzelmek száma:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="NumOfVic_value">
          <property name="font">
           <font>
            <family>DejaVu Sans</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>N/A</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="NumOfGames">
          <property name="font">
           <font>
            <family>DejaVu Sans</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Játszmák száma:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="NumOfGames_value">
          <property name="font">
           <font>
            <family>DejaVu Sans</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>N/A</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="table">
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Bingo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nyert</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Találtok</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Hossz</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Időpont</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="reset">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(165, 29, 45);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
